app/components/candidate_interface/complete_section_component.rb

```diff
@@ -1,18 +1,21 @@
 module CandidateInterface
   class CompleteSectionComponent < ViewComponent::Base
-    def initialize(application_form:, path:, request_method:, field_name:, section_review: false)
-      @application_form = application_form
+    attr_reader :section_complete_form, :path, :request_method, :review_component, :hint_text, :section_review
+
+    def initialize(section_complete_form:, path:, request_method:, review_component:, section_review: false, hint_text: false)
+      @section_complete_form = section_complete_form
       @path = path
       @request_method = request_method
-      @field_name = field_name
+      @review_component = review_component
       @section_review = section_review
+      @hint_text = hint_text
     end
 
-    def checkbox_label
-      if @section_review
-        'application_form.reviewed_checkbox'
+    def radio_button_label
```

Comment:
nitpick: would this be more descriptive as something like `completed_or_reviewed_radio_button_label` as there are two radio button labels? Bit longwinded perhaps, but might save ambiguity later?

Reply:
👍